src/background/services/paymentSession.ts

```diff
@@ -172,6 +185,15 @@ export class PaymentSession {
             }
           })
 
+          if (this.intervalInMs > 1000) {
```

Comment:
Can you add a to do item here: I feel like relying on the interval isn't really a source of truth. We can have a flag in the session that we set when we call the `adjustSessionAmount` method.

Reply:
Done. But based on what will we compute the flag? We can implement it in this one https://github.com/interledger/web-monetization-extension/issues/375